### ISSUE TITLE
This Channel is not responding

### ISSUE THREAD

**Issue by session77** [ukdtom/WebTools.bundle#482]
Using a HP Laptop running windows 7 with PMS Version 3.27.1, Web Tool Version 3.0.0, Google Chrome, any channel I try to open just shows "This Channel is not responding.  Any help would be greatly appreciated
 
![image](https://user-images.githubusercontent.com/35827050/35468406-f545d626-02ea-11e8-813b-ad4a4bed275b.png)

[PMSLogs_20180126-224352.zip](https://github.com/ukdtom/WebTools.bundle/files/1669747/PMSLogs_20180126-224352.zip)

**Comment by trumpy81** [comment]
For some reason you have a permissions error:

`Jan 26, 2018 07:24:13.388 [3528] DEBUG - Failed copying file dd_vcredist_x86_20180125184409_000_vcRuntimeMinimum_x86.log, copy_file: 5, Access is denied`

Uninstall Plex, then uninstall Microsoft Visual C++ 2015 Redistributable (x86 & x64) 14.0.24215.

When that is complete see this page: https://support.plex.tv/articles/202967086-windows-errors-trying-to-upgrade-install-or-uninstall-plex-media-server/

Then you can try installing Plex again. Do NOT run the installer 'As Administrator'.

And in future, please ask these types of questions in the Plex Windows forum, thanks.

**Comment by ukdtom** [comment]
As indicated in the Forum opening post....

`Support:
Since WebTools contains the module UAS (Unsupported AppStore) you might think, that this is the place to post questions, if a plugin installed via the UAS is not working for you, but it's not
Instead, locate the relevant support thread here
Any questions regarding other plugin's than WebTools WILL be ignored!
But.....If a plugin fails to install using WebTools UAS module, then this is the place ;)
Before you post a question, please make sure it's not already mentioned in our FAQ or Knows issues, and when posting, please add as much info as possible, as well as log files if possible, and do NOTE that we do expect that you read this thread as well, in order to not post a question already answered.`

**Comment by session77** [comment]
I uninstalled Plex, then uninstalled Microsoft Visual C++ 2015 Redistributable (x86 & x64) 14.0.24215. Then I reinstalled Plex and when I tried to install WebTools I got and error that this program might not have installed correctly see pic.  I'm not sure if the download I'm selecting is the correct one see pic below this is where I'm downloading webtools for windows 7.  Any help will be greatly appreciated 

![webtools not compatible with version of windows](https://user-images.githubusercontent.com/35827050/35527207-e9b23104-0521-11e8-94cb-b0004376d19c.jpg)

![webtools download location](https://user-images.githubusercontent.com/35827050/35527389-7fa8ea72-0522-11e8-9f73-7371c535461f.jpg)

**Comment by ukdtom** [comment]
It's the right one, but as @trumpy81 said, this is not related to WebTools, but instead to your Windows installation....

**Comment by session77** [comment]
I uninstalled Plex, then uninstalled Microsoft Visual C++ 2015 Redistributable (x86 & x64) 14.0.24215. Then I reinstalled Plex and when I tried to install WebTools I got and error that this program might not have installed correctly see pic. I'm not sure if the download I'm selecting is the correct one see pic below this is where I'm downloading webtools for windows 7. Any help will be greatly appreciated

![webtools not compatible with version of windows 2](https://user-images.githubusercontent.com/35827050/35527961-4f58546e-0524-11e8-8bca-649902c4ec3b.jpg)

![webtools download location](https://user-images.githubusercontent.com/35827050/35528002-61554186-0524-11e8-8a47-b3287fddadf2.jpg)
